clients/cloud/csharp/Program.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Confluent.Kafka;
+using Confluent.Kafka.Admin;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace CCloud
+{
+    class Program
+    {
+        static string ExtractJaasValue(string jaasConfig, string key)

```

Comment:
docstrings what this does (translates the java jaas config for plain auth to .NET client conf properties)

Reply:
fair enough